charmhelpers/core/hookenv.py

```diff
@@ -370,6 +370,12 @@ def config(scope=None):
     """
     global _cache_config
     config_cmd_line = ['config-get', '--all', '--format=json']
+    try:
```

Comment:
This looks a little ugly:  Why not just do

```python
exc_json = json.decoder.JSONDecodeError if six.PY3 else ValueError
```

Reply:
Indeed, however it seems like the exception thrown by json changed from ValueError to JSONDecodeError between Python3.4 and Python3.5.

The PY3 test suite run on Travis uses Python3.4

Thus a binary test for PY2 vs. PY3 is not enough.